examples/vnet/doca_gw_dpdk.c

```diff
@@ -1201,12 +1205,12 @@ struct doca_gw_port * doca_gw_dpdk_port_start(struct doca_gw_port_cfg *cfg,
 {
 	struct doca_gw_port *port = doca_alloc_port_byid(cfg->port_id, cfg);
```

Comment:
This is not correct.
user gives you the port id (dpdk use case), in cfg-.devargs, you should take that as the port id.

Reply:
yes, need devargs, i suggest to do atoi translate in API layer, dpdk layer only use port_id.